onnx/onnx_pb.h

```diff
@@ -7,7 +7,7 @@
 #ifdef ONNX_ML
 #include "onnx/onnx-ml.pb.h"
 #else
-#include "onnx/onnx.pb.h"
+#include "onnx.pb.h"
```

Comment:
Quick check if this is intentional. Seems that we might want to indeed to onnx/onnx.pb.h?

(If you made this change due to protobuf generated files, please ping back - I can help with that)

Reply:
Aha, you're right, I made this change due to files generated by protoc were put directly in build folder. Didn't figure out a way here to tell PROTOBUF_GENERATE_CPP to put generated files in a specific folder :(

Reply:
Yeah, so protobuf's default command does not allow that, which is a big bummer. You can use a custom command with something like this:

https://github.com/caffe2/caffe2/blob/4f534fad1af9f77d4f0496ecd37dafb382330223/cmake/ProtoBuf.cmake#L137-L184

Note that if you want to use onnx as Windows dlls, you are going to need the dllexport_decl definition because otherwise protobuf does not work well with dlls.

Reply:
Thank you Yangqing! Yep, that's my origin change to use a customized generation function. OK, changed to the way.